src/app/account-transfers/make-account-transfers/make-account-transfers.component.ts

```
@@ -106,25 +102,33 @@ export class MakeAccountTransfersComponent implements OnInit, AfterViewInit {
+  this.accountType = this.route.snapshot.queryParams['accountType'];
+  switch (this.accountType) {
+    case 'fromloans':
+      this.accountTypeId = '1';
+      this.id = this.route.snapshot.queryParams['loanId'];
+      break;
+    case 'fromsavings':
+    case 'interbank':
+      this.accountTypeId = '2';
+      this.id = this.route.snapshot.queryParams['savingsId'];
+      this.interbank = this.route.snapshot.queryParams['interbank'] === 'true';
+      const navigationBalance = this.router.getCurrentNavigation()?.extras?.state?.balance;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)